packages/visx-axis/src/axis/AxisRenderer.tsx

```diff
@@ -89,13 +93,13 @@ export default function AxisRenderer<Scale extends AxisScale>({
           className={cx('visx-axis-label', labelClassName)}
           {...getLabelTransform({
             labelOffset,
-            labelProps,
+            labelPropsDefault,
```

Comment:
Looks like TS is failing because this is a named argument

```suggestion
            labelProps: combinedLabelProps,
```